src/Uno.UI/UI/Xaml/Controls/Icon/IconElement.macOS.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Windows.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AppKit;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class IconElement
```

Comment:
Indentation

Reply:
Reformated :-)

Reply:
Still incorrect :-S

Reply:
Looks like vs4mac is not handling the .editorconfig file properly. Maybe it's placed at the wrong location.

Reply:
Fixed, Parallels' Visual Studio 2019 is more and more useful :-D .